helm-charts/mlflow-tracking/values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for mlflow-tracking-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: public.ecr.aws/data-on-eks/mlflow
```

Comment:
How do we ensure this image is latest from mlflow? Can you also add a folder with file `docker/Dockerfile-mlflow` to the root of the repo and add deployment instructions to the same Dockerfile in the header ?